Project 1: Copy of New York Page

GorgiMitev opened the pull request:
Made the required changes after the code review-regarding positioning and styling of certain elements

GorgiMitev committed:
changed the readme file

GorgiMitev committed:
finshed the required html for the form element

GorgiMitev committed:
finished the styling of the form element

GorgiMitev committed:
made required styling changes

GorgiMitev committed:
correcte the last two stylint errors

GorgiMitev committed:
rectified the last errors of the html file

GorgiMitev committed:
made the last required changes of the project

GorgiMitev committed:
made the last required changes of the project

zilton7 commented on index.html:
```diff
@@ -371,6 +368,42 @@ <h3>A Special Time</h3>
+        <label class="label-tab" for="email">Email</label><br>
+        <input class="input-tab" type="text" id="email" name ="email" maxlength="20" placeholder="Enter your email" required><br>
+        
+        <label class="label-tab" for="age">Age</label><br>
```
This wasn't required, but nice to be added

Forison reviewed (changes requested):
## Status::Changes Required❌

# 🙋‍♂️ @Solenka1 & @zilton7 

#### Good job team on adding the link of the previous branch whilst creating the pull request, and good job on addressing some of the issues raised in the previous review. 
This review will be based on the last two reviews of the old P/R. Most of the issues were address however here are few we have missed and the issue of the horizontal scroll

* [ ] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [ ] All images in the Readme should be functional. Currently the image of the project is not showing, let address it.
* [ ] Let us, address the horizontal scroll that is found on the page, this is likely due to an element overflowing the general width of the page.
![Screenshot from 2021-02-18 13-10-44](https://user-images.githubusercontent.com/29695661/108401911-de40b280-71ea-11eb-99f9-340df8360160.png)

**[OPTIONAL]**
* [ ] It is better to give a height auto to an image, to prevent it from strecthing.
* [ ] t has a fixed footer (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) 

#### That is it kindly make the changes and submit the project for a re-review

#### Happy coding

Addo Forison

GorgiMitev committed:
made all the required changes after the code review

GorgiMitev committed:
made last styling adjustments

GorgiMitev committed:
added finishing touches to the readme file

GorgiMitev committed:
Updated the readme file

made changes of the readme file

BrianSammit reviewed (changes requested):
CHANGES REQUIRED :recycle: :red_circle: 
------------------
Hello team!! :wave: you did an amazing job on this project :100: Job job working on the required changes :muscle: 
There are a few changes to make to full fit this project :hammer_and_wrench: see them below :eyes: :point_down: 
---------------

- [ ] Improve your README.md file. A professional readme makes your repository looks professional that going to help you to land your first job. See the improvements to make below :point_down: 

- Write a brief description of the project, E.g "This project is a clone from the New York Times page that we made to learn about ......." 
- In "Getting Started" you have a misspelling `nstructions` check your grammar with Grammarly before you send it for review. 
- In "Getting Started" you write "To get a local copy up and running follow these simple example steps." but you don't have any step on how to copy the repository. 
- In "Prerequisites" you have to write the minimum requisites to run the repository on the local machine E.g (HTML, CSS, ...) write the requisites on bullet points to make your Readme looks professional. 

<img width="603" alt="descriptive_readme" src="https://user-images.githubusercontent.com/64173189/108527103-e3ac0480-729f-11eb-8dcd-3a610a97c950.png">

- [ ] Write better commits. Remember that you must write your commit messages in the imperative. [Here](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices#formatting-rules) you can read about how to write an excellent commit message. [Here](https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message) you can read how to changes commit messages. You don't need to changes the messages this time but keep the formatting rules in mind for the next commits. 
- [ ] Improve your pull request description. Not just write that you made the required changes, write what changes you make, or write what the pull-request contain. Remember that this description is to help (in this case) the reviewers to know what is in the pull request. In your case, I can see that you are changing the description every time you made a change, you can add new messages in the pull request without changing the description of the pull request. 
-----------------
Make these required changes :point_up: and sen it for another review. :muscle: 
HAPPY CODDING!!
![AngelicConcreteHypsilophodon-max-1mb](https://user-images.githubusercontent.com/64173189/108531578-d6dddf80-72a4-11eb-83f6-57d0cde8bf36.gif)

GorgiMitev committed:
IMPROVED THE README FILE

- improved the spelling of the file
- improved the getting started section
- added an explanation regarding the project

wathigo reviewed (approved):
## APPROVED :heavy_check_mark: 
Great job implementing the changes pointed out by the previous TSE(s).

**You may proceed to the next project.**

**Happy Learning :student: 
Simon Wathigo**
